python/tskit/trees.py

```diff
@@ -833,6 +833,57 @@ def unrank(rank, num_leaves):
         """
         return combinatorics.RankTree.unrank(rank, num_leaves).to_tsk_tree()
 
+    def count_topologies(self, sample_sets=None):
+        """
+        Calculates the distribution of embedded topologies for every combination
+        of the sample sets in ``sample_sets``. ``sample_sets`` defaults to all
+        samples in the tree grouped by population. In this case, the topologies
```

Comment:
do the sample sets have to be nonoverlapping? (this affects the math below)

Reply:
Yes they do, good point.